Jenkinsfile.conda

```diff
@@ -0,0 +1,56 @@
+pipeline {
+    agent any
+    environment {
+        idl_version = "${GIT_TAG_NAME}"
+        xml_version = "4.6.0"
+        dockerImageName = "lsstts/conda_package_builder:latest"
+        container_name = "idl_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
+    }
+
+    stages {
+        stage("Pull Docker Image") {
+            steps {
+                script {
+                    sh """
+                    docker pull ${dockerImageName}
+                    """
+                }
+            }
+        }
+        stage("Create IDL Conda package") {
+            steps {
+                script {
+                    sh """
+                    docker run --name ${container_name} -di --rm --env TS_XML_VERSION=v${xml_version} --env IDL_VERION=${idl_version} \
```

Comment:
There is a typo in `IDL_VERION`. Also, if you are passing `idl_version` as `IDL_VERSION`, make sure you update the `git_rev: {{ idl_version }}` in the `meta.yaml` file to be `git_rev: {{ IDL_VERSION }}`.

Reply:
Fixed in commit 0d4c793dcf30013c598f6d415f561951410c47bf